lib/ts/recipe/dashboard/api/userroles/roles/deleteRole.ts

```diff
@@ -8,9 +9,23 @@ const deleteRole = async (
     ___: string,
     options: APIOptions,
     __: any
-): Promise<{
-    status: "OK" | "ROLE_DO_NOT_EXISTS";
-}> => {
+): Promise<
+    | {
+          status: "OK";
+          didRoleExist: boolean;
+      }
+    | {
+          status: "FEATURE_NOT_ENABLED_ERROR";
+      }
+> => {
+    try {
+        UserRolesRecipe.getInstanceOrThrowError();
+    } catch (_) {
+        return {
+            status: "FEATURE_NOT_ENABLED_ERROR",
```

Comment:
i thought you are not going to do this in any API since it's done in the entry point API?